forms-flow-components/src/components/CustomComponents/Radio.tsx

```diff
@@ -29,7 +31,7 @@ export const CustomRadioButton: React.FC<CustomRadioButtonProps> = ({
           id={`inline-radio-${index + 1}`}
           data-testid={`${dataTestid}-inline-radio-${index + 1}`}
           key={`radio-${index + 1}`}
-          defaultChecked={index === 0}
+          defaultChecked={index === indexValue}
```

Comment:
onchange is common handler for selection

Reply:
class name is empty

Reply:
where is value prop in this radio btton

Reply:
instead of passing indexValue do we need to pass the value to make sure default check. @arun-s-aot

Reply:
Agree to that. index checking can be replaced with value checking @Bonymol-aot